index.html

```diff
@@ -71,7 +104,7 @@ <h2 class="tonic">Tonic</h2>
                 <li> <img src="images/Tag(2).png" alt="Tags"> </li>
               </ul>
             <div class="Job1-btn">
-              <button type="button" name="button" class="Job1-button"> See Project</button>
+              <button type="button" name="button" class="Job1-button btn"> See Project</button>
```

Comment:
[OPTIONAL]

- The hover state background color of your  `See Project` button is currently red. please use the background color specified in tour chosen Figma design template👍 

![image](https://user-images.githubusercontent.com/63473584/180338124-a228b5ea-6a0e-4cec-8bcd-5c177a4abd9e.png)

FIgma Design 👇 

![image](https://user-images.githubusercontent.com/63473584/180338485-e632761d-4305-4615-8697-90b860a1544f.png)